README.md

```diff
@@ -248,19 +155,16 @@ before the deployment is considered successful.
   that use the `RollingUpdate` strategy.
   - Percent (e.g. 90%): The deploy is successful when the number of new pods that are ready is equal to
   `spec.replicas` * Percent.
-- `krane.shopify.io/prunable`: Allows a Custom Resource to be pruned during deployment.
-  - _Compatibility_: Custom Resource Definition
-  - `true`: The custom resource will be pruned if the resource is not in the deploy directory.
-  - All other values: The custom resource will not be pruned.
 - `krane.shopify.io/predeployed`: Causes a Custom Resource to be deployed in the pre-deploy phase.
   - _Compatibility_: Custom Resource Definition
   - _Default_: `true`
   - `true`: The custom resource will be deployed in the pre-deploy phase.
   - All other values: The custom resource will be deployed in the main deployment phase.
 
+
 ### Running tasks at the beginning of a deploy
 
-To run a task in your cluster at the beginning of every deploy, simply include a `Pod` template in your deploy directory. `kubernetes-deploy` will first deploy any `ConfigMap` and `PersistentVolumeClaim` resources in your template set, followed by any such pods. If the command run by one of these pods fails (i.e. exits with a non-zero status), the overall deploy will fail at this step (no other resources will be deployed).
+To run a task in your cluster at the beginning of every deploy, simply include a `Pod` template in your deploy directory. `krane` will first deploy any `ConfigMap` and `PersistentVolumeClaim` resources in your template set, followed by any such pods. If the command run by one of these pods fails (i.e. exits with a non-zero status), the overall deploy will fail at this step (no other resources will be deployed).
```

Comment:
'template set' is more of an internal detail of `krane` as opposed to a top level concern. From a user's point of view, they are simply passing in a list of filenames/dirs. It might be more accurate to say `resources from files provided to krane`.

:man_shrugging: I'm not too hung up on it, though